frontend/src/metabase/query_builder/components/chart-type-selector/viz-order.ts

```diff
@@ -0,0 +1,21 @@
+import type { CardDisplayType } from "metabase-types/api";
+
+export const DEFAULT_ORDER: CardDisplayType[] = [
```

Comment:
nit: rename to `DEFAULT_VIZ_ORDER` so that it's a bit easier to search for / import?

Reply:
done